src/apis/NotificationApi.ts

```diff
@@ -33,5 +33,19 @@ export interface NotificationApi {
    * @param notificationId the id of the Notification to retrieve.
    * @return a Promise containing the Notification or undefined if something goes wrong.
    */
-  getNotification(notificationId: string): Promise<Notification | undefined>
+  getNotification(notificationId: string): Promise<Notification | undefined>;
+
+  /**
+   * Gets all the Notifications with type NEW_USER_OWN_DATA_ACCESS and status "pending" that the current dataOwner can access
+   * @return an Array of the Notifications matching those criteria
+   */
+  getPendingNotificationsFromNewUsers(): Promise<Array<Notification>>;
```

Comment:
You can rename in `getPendingNotifications` and return all types of them. The user will treat them one by one afterwards.